src/pages/brownBag/OngoingBrownBag.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../styles/base/_variables.scss';
+
+.onlistBrownBag {
```

Comment:
it's more preferable to have css class names separated by dashes or underscores rather than camel case.

Reply:
That won't be possible since I am using className={scss file.class} to reference the class. using a dash makes the word seem separated after the underscore which throws an error.
I think this has been changed with the new version of React where you can now use `class` instead of `className`

Reply:
Ooh! I get it now, makes more sense.